output/missing-source-info.ts

```diff
@@ -1610,6 +1627,23 @@ const missingSources: { [key: string]: number[] } = {
     3939809874, // Plate of Transcendence
     4177973942, // Vest of Transcendence
   ],
+  gotd: [
```

Comment:
Interesting, I didn't realize that the aliases cause the hashes to be repeated. Feels like an opportunity?

Reply:
Agreed - filed #507.